### PR TITLE
Fixes #7493

### LINKED ISSUE #7493: round-start salicyclic acid pills are over the OD threshold
The pills of salicyclic acid which are inside medkits at the start of the round. They contain 50u of salicyclic acid.
The OD threshold for sali-acid is 25.

If you eat just one pill, it will hurt you. This is probably unintentional.

I've verified this by grinding the pills (produces 50u) and by popping a single pill and scanning myself to observe the rising brute damage.

On a tangentially-related note, fire first aid kits contain these pills when they probably shouldn't, given that sali-acid heals brute and not burn.

### PR DESCRIPTION
**Pull request by Iamgoofball**
Fixes #7493

this also fixes ODing on epi pills

makes roundstart pills append their reagent number to their name as per request, only roundstart ones do this

### CODE REVIEW

**Comment by Deantwo** [comment]
:+1:

**Comment by paprka** [comment]
Why can you OD on these chems anyway, they're helpful chems and not superpower ones

**Comment by Deantwo** [comment]
Yeah, we need that handheld reagent scanner.
It could be made to detect if a chemical is in an overdosing amount of it.

**Comment by phil235** [comment]
@Iamgoofball Can you make your PR titles more explicit please. Something like "Fixes round-start salicyclic acid pills" instead of just an issue number.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -129,4 +129,4 @@
 	name = "salicyclic acid pill"
```
Comment (Deantwo) [inline code/modules/reagents/reagent_containers/pill.dm]: Can you please make it ```Salicylic acid pill (24u)``` so the name matches that of chemist created pills?

PS: The name is spelled wrong. Fixed that above.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -111,7 +111,7 @@
 	name = "epinephrine pill"
 	desc = "Used to stabilize patients."
 	icon_state = "pill5"
```
Comment (Deantwo) [inline code/modules/reagents/reagent_containers/pill.dm]: Can you please make it ```epinephrine pill (15u)``` so the name matches that of chemist created pills?

**Comment by Deantwo** [comment]
Do that for all the pills please while your at it.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -111,7 +111,7 @@
 	name = "epinephrine pill"
```
Comment (Deantwo) [inline code/modules/reagents/reagent_containers/pill.dm]: Can you please make it ```epinephrine pill (15u)``` so the name matches that of chemist created pills?

**Comment by Deantwo** [comment]
Do that for all the pills please while your at it.

**Comment by Iamgoofball** [comment]
@paprka because it does a shitton more then "stops losebreath" now and without overdosing I can basically just rapidfire inject 200 medipens and keep it all sitting in my internal reagents so that if I go into crit in the next hour or so I stay stabilized

OD is the solution to "inject 100 of every beneficial chem and call it good"

**Comment by Deantwo** [comment]
Interesting way of doing it, it should work fine.

**Pull request merged**